Libraries/acdriver/Headers/acdriver/Compile/Output.h

```diff
@@ -91,6 +109,12 @@ class Output {
     ext::optional<std::string> const &launchImage()
     { return _launchImage; }
 
+    /*
+     * If an image with a non-standard format should be compile.
```

Comment:
This sounds like an incomplete thought -- it's not a bool (which I associate with "if"), it's a list.  Maybe change to something like "List of non-standard image formats that are allowed

Reply:
You are right, I change the plans here and forgot to update the comments.